contracts/RMRK/base/RMRKBaseStorage.sol

```diff
@@ -53,7 +46,7 @@ contract RMRKBaseStorage is IRMRKBaseStorage {
         _;
     }
 
-    function _onlySlot(uint64 partId) internal view {
+    function _onlySlot(uint64 partId) private view {
```

Comment:
Shouldn't this be available in the contracts that inherit this one?

Reply:
Do you see any reason? 
I think the modifier should be enough, no need to call this directly. This is actually put into a separate function just to save some contract size.

Reply:
I could see some use cases where people would want to use it in their own business logic, but on the other hand, if they need it, they can implement it themselves